reference/channels.v3.yml

```
@@ -1933,6 +1937,12 @@ components:
         title: A deleted channel cannot be updated. Channel can be restored within 90 days after deleting by contacting BigCommerce support team
         type: /api-docs/getting-started/api-status-codes
         errors: {}
+    error_422_invalid_channel_name_resp_example:
+      value:
+        status: 422
+        title: Only Latin alphabet letters, Arabic numerals, dashes, and underscores are allowed in the channel name.
```

```suggestion
        title: Only latin alphabet letters, arabic numerals, dash and underscore are allowed to be used as channel name.
```

restoring actual error message.